### ISSUE TITLE
Undefined offset when batch copy parent menu and not child menu

### ISSUE THREAD

**Issue by Quy** [joomla/joomla-cms#19547]
### Steps to reproduce the issue
Install the last demo
Under Menus, select `Main Menu`
Select `Site Map` & `Sample Sites`
Click `Batch` button
Add to `Australian Parks` menu
Click `Copy`
Click `Process` button
See PHP error log

### Expected result
no notices

### Actual result
```
PHP Notice:  Undefined offset: 290 in \libraries\src\MVC\Model\AdminModel.php on line 284
PHP Notice:  Undefined offset: 439 in \libraries\src\MVC\Model\AdminModel.php on line 284
PHP Notice:  Undefined offset: 445 in \libraries\src\MVC\Model\AdminModel.php on line 284
PHP Notice:  Undefined offset: 446 in \libraries\src\MVC\Model\AdminModel.php on line 284
```

### System information (as much as possible)
v3.8.3 & v3.8.4

### Additional comments
These notices are generated when batch copy the parent menu item and not child menu items.

**Comment by sandewt** [comment]
I can NOT reproduce this issue.

Joomla! 3.8.4
PHP Version 7.1.9
Xampp
Windows 10
Path to the _error.php_ log folder ..\JOOMLA_3.8.4-2\Joomla\administrator/logs

<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19547">issues.joomla.org/tracker/joomla-cms/19547</a>.</sub>

**Comment by Quy** [comment]
Please check PHP error log and not Joomla error log.

**Comment by Quy** [comment]
When the menu item has child menu items, then these notices are generated.

**Comment by sandewt** [comment]
@Quy , I understand it.

YES, I can reproduce this issue.

Path to the PHP log folder C:\xampp\apache\logs\error.log

<code>
PHP Notice:  Undefined offset: 290 in C:\\xampp\\htdocs\\JOOMLA_3.8.4-2\\Joomla\\libraries\\src\\MVC\\Model\\AdminModel.php on line 284
</code>
and so on .

**Comment by ghost** [comment]
closed as having Pull Request #19587
